performance-tools/performance-cluster/chart/kyma-cluster/templates/pod.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: setup-{{ template "kyma-cluster.fullname" . }}
+  labels:
+    app: {{ template "kyma-cluster.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    grade: "{{ .Values.labels.grade}}"
+    "kyma-project.io/kyma-cluster": "{{ .Values.labels.grade}}"
+spec:
+  serviceAccountName: setup-{{ template "kyma-cluster.fullname" . }}
+  restartPolicy: Never
+  containers:
+  - name: kyma-cluster
+    image: "{{ .Values.containerRegistry.path }}/{{ .Values.image.dir }}performance-cluster:{{ .Values.image.version }}"
+    imagePullPolicy: "{{ .Values.image.pullPolicy}}"
+    command: ["/scripts/kyma-gke-cluster.sh"]
+    env:
+    - name: APP_MAX_CONCURRENCY_LEVEL
```

Comment:
Prow specific variables are follows, we may not need them:
- APP_MAX_CONCURRENCY_LEVEL
- APP_LOGGER_LEVEL
- DOCKER_IN_DOCKER_ENABLED